lib/blueprinter/base.rb

```diff
@@ -1,12 +1,13 @@
 require_relative 'blueprinter_error'
-require_relative 'helpers/active_record_helpers'
+require_relative 'configuration'
```

Comment:
This is already required in https://github.com/procore/blueprinter/blame/master/lib/blueprinter.rb#L1 so this is not needed.

Reply:
This means that we cannot require `blueprinter/base.rb`, since it uses `Blueprinter.configuration`, without requiring `blueprinter.rb`. Is this the intended behavior? Should the `configuration` require be only in base, insetad of in `blueprinter.rb`?

Reply:
I see your point. Yea we can leave configuration here, just remove it from blueprinter.rb and you're good to go!